.cargo/config.toml

```
@@ -6,3 +6,7 @@ features = ['all']
 
 # This is needed so the "include" statement above works.
 config-include = true
+
+[unstable.git]
+shallow_index = true
+shallow_deps = true
```

Manifest keys are kebab-case:

```suggestion
shallow-index = true
shallow-deps = true
```

---

I just tried with these keys, they don't enable shallow cloning on nightly whereas using the keys with underscores works (I can see the checkouts having the `-shallow` suffix).

I based the configuration on this comment: https://github.com/rust-lang/cargo/issues/13285#issuecomment-2024346828

---

Indeed! That's a bit weird, I suppose that would be changed before it's stabilized? Disregard my suggestion then :slightly_smiling_face: